src/lib/ChatMessage/ChatMessage.svelte

```
@@ -136,7 +154,16 @@
         <div class="header">{@render header()}</div>
       {/if}
 
-      <div class="bubble">
+      <div
+        class="bubble"
+        class:clampable={clampLines > 0}
+        data-clamped={clampActive ? 'true' : undefined}
```

ESLint

[error] undefined is not allowed. Use null or proper type checking instead.

(no-restricted-syntax)